Cargo.toml

```diff
@@ -1,6 +1,6 @@
-[package]
-name = "dependobuf"
-version = "0.1.0"
-edition = "2021"
+[workspace]
+members = ["ast"]
```

Comment:
I'd name the main crate "dbuf-core", it's going to contain compilation stages which are used by both compiler and lsp